src/frontend/js/components/AddressesManagement/index.spec.tsx

```diff
@@ -472,7 +473,7 @@ describe('AddressesManagement', () => {
     expect($saveField).toBeNull();
 
     // - But finally user cancels his action
-    const $cancelButton = screen.getByRole('button', { name: 'Cancel edition', exact: true });
+    const $cancelButton = screen.getByRole('button', { name: 'Cancel', exact: true });
```

Comment:
Is it related to this commit purpose ?

Reply:
It is not related to the userEvent update but this test fails since dependencies udpate.